src/Stripe.net/Services/Orders/OrderService.cs

```diff
@@ -69,6 +69,16 @@ public virtual Task<Order> PayAsync(string orderId, OrderPayOptions options = nu
             return this.RequestAsync(HttpMethod.Post, $"{this.InstanceUrl(orderId)}/pay", options, requestOptions, cancellationToken);
         }
 
+        public virtual OrderReturn Return(string orderId, OrderReturnOptions options = null, RequestOptions requestOptions = null)
```

Comment:
In other languages like PHP the method is named `returnOrder` instead. I think the short name is better if supported in dotnet but also flagging to @ob-stripe

Reply:
Good call out. I followed the other custom method in this file called `Pay` but happy to name it whatever.

Reply:
👍  for `Return`